frontend/integration-tests-cypress/cypress-e2e/cypress/support/commands.js

```diff
@@ -0,0 +1,72 @@
+// ***********************************************
```

Comment:
Remove the "example" comment as you have customized it?

Reply:
fixed